src/components/app/data/hooks/useDefaultSearchFilters.js

```diff
@@ -7,11 +7,13 @@ import {
 } from '@edx/frontend-enterprise-catalog-search';
 import useEnterpriseCustomer from './useEnterpriseCustomer';
 import useSearchCatalogs from './useSearchCatalogs';
+import { features } from '../../../../config';
 
 export default function useDefaultSearchFilters() {
   const { refinements, dispatch } = useContext(SearchContext);
   const showAllRefinement = refinements[SHOW_ALL_NAME];
   const { data: enterpriseCustomer } = useEnterpriseCustomer();
+  const videoSearchQuery = features.FEATURE_ENABLE_VIDEO_CATALOG ? '' : ' AND (NOT content_type:video)';
```

Comment:
nit: is it worth adding a `FIXME` type comment similar to the other code path we modified to make it more evident that this is a temporary stop-gap solution due to inconsistent content metadata returned by Algolia?